composer/utils/file_helpers.py

```diff
@@ -315,7 +315,8 @@ def parse_uri(uri: str) -> Tuple[str, str, str]:
                               Backend and bucket name will be empty string if the input is a local path
     """
     parse_result = urlparse(uri)
-    backend, bucket_name, path = parse_result.scheme, parse_result.netloc, parse_result.path
+    backend, net_loc, path = parse_result.scheme, parse_result.netloc, parse_result.path
```

Comment:
Do you need this logic in `maybe_create_remote_uploader_downloader_from_uri` too?

Reply:
isn't it in both? they both use that function

Reply:
ah sorry, i misread which function this was in